### PR TITLE
fix: default reminder ui bug

### PR DESCRIPTION
**Pull request by abdou6666**
#6185 

before : 
image

after : 
image

### CODE REVIEW

**Comment by hamza221** [comment]
The text is too long, it will never be displayed fully.  @ChristophWurst would only changing the default reminder be enough to close the issue ?

File: src/components/AppNavigation/Settings.vue [hunk 1]
```diff
@@ -256,8 +255,9 @@ export default {
 		},
 		defaultReminderOptions() {
 			const defaultAlarms = getDefaultAlarms().map(seconds => {
+				const label = seconds === 0 ? t('calendar', "at event's start") : t('calendar', "{name} before the event's starts", { name: moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize() })
```
Comment (miaulalala) [inline src/components/AppNavigation/Settings.vue:258]: Too much text for the width of the field unfortunately.

```suggestion
				const label = seconds === 0 ? t('calendar', "At event start") : moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize()
```

**Review by miaulalala — CHANGES_REQUESTED** [review]

File: src/components/AppNavigation/Settings.vue [hunk 2]
```diff
@@ -256,8 +255,9 @@ export default {
 		},
 		defaultReminderOptions() {
 			const defaultAlarms = getDefaultAlarms().map(seconds => {
+				const label = seconds === 0 ? t('calendar', "At event start") : moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize()
```
Comment (miaulalala) [inline src/components/AppNavigation/Settings.vue:258]:
```suggestion
				const label = seconds === 0 ? t('calendar', 'At event start') : moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize()
```

CI doesn't like that :see_no_evil: 

If you could change that then we can merge - thanks so much!

**Review by miaulalala — CHANGES_REQUESTED** [review]

**Review by hamza221 — APPROVED** [review]

**Review by miaulalala — APPROVED** [review]

**Comment by miaulalala** [comment]
No backport due to translations

**Pull request merged**